pkg/controller/job/job_controller_test.go

```diff
@@ -6594,19 +6594,21 @@ func TestWatchOrphanPods(t *testing.T) {
 					sharedInformers.Batch().V1().Jobs().Informer().GetIndexer().Delete(tc.job)
```

Comment:
I wonder if this test is leaking routines. We are not canceling the parent context, which is used for `go.manager.Run`

Reply:
Please address

Reply:
I've added a t.Cleanup when the context is created